### DISCUSSION TITLE
Cancellation without user interaction

### DISCUSSION THREAD

**Discussion by yesnomaybewtf** [derhansen/sf_event_mgt#1226] [General]
Hi,
first of all, i think its not an issue with sf_event_mgt,  i just hope to find other people here with the same problem.
We are using typo3 9.5 (LTS) and plugin version 4.5.0.
Some people have the following problem: they sign up for an event, get the mail where they have to confirm the signup, an without any interaction, the signup is cancelled.
I have a suspicion that some kind of malware/virusscanner is checking the urls in the mail in the background, and so the cancellation process is starting.

So, is there anybody out there with the same kind of issue? :)

**Comment by derhansen** [comment]
Yes, this has already been discussed - see: https://github.com/derhansen/sf_event_mgt/issues/1179

**Reply by yesnomaybewtf** [reply]
Thanks! Got to talk to my developers.
